test/view.test.ts

```diff
@@ -0,0 +1,45 @@
+import { beforeEach, describe, expect, it } from "./suite.ts";
+
+import { seedrandom } from "../src/seedrandom.ts";
+import { createGraph, normal, GraphView, createView } from "../mod.ts";
+
+describe("view", () => {
+  let view: GraphView;
+  let seed = seedrandom("normal.test.ts");
+
+  beforeEach(() => {
+    const graph = createGraph({
+      seed,
+      types: {
+        vertex: [{
+          name: "User",
+          relationships: [{
+            type: "User.repositories",
+            direction: "from",
+            size: normal({ mean: 3, standardDeviation: 2 }),
+          }],
+        }, {
+          name: "Repository",
+          relationships: [],
+        }],
+        edge: [{
+          name: "User.repositories",
+          from: "User",
+          to: "Repository",
+        }],
+      },
+    });
+
+    view = createView(graph);
+  });
+
+  it("has Users type", () => {
+    expect(view.User).toBeDefined();
+  });
+
+  it("has Repository type", () => {
+    expect(view.Repository).toBeDefined();
+  });
+
+  // TODO: how do I read generated objects from this?
```

Comment:
First, you have to create a vertex, and then make sure that the vertex shows up in the view with its properties as well as its relationships: e.g.

```ts
let user = createVertex(graph, 'User', { 
  name: 'thefrontside',
  'User.repositories': [{
    name: 'graphgen', 
  }]
});
expect(view.User[user.id].attributes).toEqual({ name: 'thefrontside' });
let [repo] = Object.values(view.User[user.id].relationships);
expect(repo.attributes).toEqual({ name: 'graphgen'})
```